### PR TITLE
Log Middleware

### PR DESCRIPTION
**Pull request by jdart**
I added a simple middleware to output logging to aid in debugging, and I also changed one of the example .env variables that was wrong.

### CODE REVIEW

**Review by ryanhofer — COMMENTED** [review]
Looks good.

The only change I would suggest is to call `app.use(log)` at the top instead of attaching the middleware to every route individually.

**Comment by jdart** [comment]
I've updated with your suggestion @ryanhofer

**Review by ryanhofer — APPROVED** [review]

File: src/app.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ app.get('/oauth/redirect', (req, res) => {
         'provide_shipping_rates',
     ].join(' ');
 
+    console.log('Scopes: ', scope);
```
Comment (grandelc) [inline src/app.js:38]: Was this console log intended to be committed? If not, we could perhaps remove it

Comment (jdart) [reply src/app.js]: Good catch.. removed.

**Comment by grandelc** [comment]
@jdart This looks good to me 👍

**Review by nathankoop — APPROVED** [review]

**Pull request merged**